src/Components/Dropdowns/Composed/MenuDropdown.tsx

```diff
@@ -25,6 +25,7 @@ import {
   StandardFunctionProps,
 } from '../../../Types'
 import classnames from 'classnames'
+import '../ScrollBarStyles.scss'
```

Comment:
This adds styles to the List component so that it matches the Clockface scrollbar styles.

Reply:
@wdoconnell this will fix this weird scrollbar you saw. 👍 

![image](https://user-images.githubusercontent.com/18511823/177014253-806f5dd0-ca7e-4a43-971f-1faf659e5cf1.png)

Reply:
after the styles applied:

<img width="486" alt="Screen Shot 2022-07-02 at 12 40 46 PM" src="https://user-images.githubusercontent.com/18511823/177014269-80761008-4e85-43b4-b709-fbc4d4de6db6.png">

Reply:
@ChitlangeSahas Thanks for fixing the styles!